qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -405,6 +407,7 @@ def __init__(
         envelope: Optional[sym.Expr] = None,
         constraints: Optional[sym.Expr] = None,
         valid_amp_conditions: Optional[sym.Expr] = None,
+        disable_validation: Optional[bool] = False,
```

Comment:
I understand this mirrors implementation of the amplitude limit, but I don't really like the implementation because this makes me think `disable_validation` is an instance variable. To avoid confusion, this must be dropped from the constructor. 

This is another concern; when you set the flag of the `SymbolicScalablePulse`, does this change the behavior of the `SymbolicPulse` instance? I think only opposite pattern happens thanks to MRO, which might be also confusing to users. I don't think we need to implement complicated logic now (because we can later make it compiler pass after #10759 ). At least we need to update class documentation about how we disable the validation, e.g.

```python
SymbolicPulse.disable_validation = True
my_pulse = Gaussian(duration=10, amplitude=1.23, sigma=2)
```

I don't think instance-wide controllability is necessary. For dynamics users the validation subroutine shouldn't get called for every pulse to be JAX compatible. This might be also preferred by users who concern performance.

Reply:
Thanks.